Client/Frontend/Strings.swift

```diff
@@ -90,6 +90,11 @@ extension String {
                 tableName: "BiometricAuthentication",
                 value: "Authenticate to access passwords.",
                 comment: "Biometric authentication is when the system prompts users for Face ID or fingerprint before accessing protected information. This string asks the user to enter their device passcode to access the protected screen.")
```

Comment:
Seems like we won't be using the old string at all then. Should we remove it in this PR?

Reply:
In theory we won't be using it, so this is what I don't get then why do we even need a V2 😅 
We can update v116 and that should delete and trigger a new string change no?

BUT, v115 which is currently not out will lose this string so lets keep it for now